www/templates/html/Media.tpl.php

```diff
@@ -46,11 +46,18 @@
 $controller->setReplacementData('head', $meta);
 $getTracks = $context->getTextTrackURLs();
 
+$ratioClass = 'class="dcf-ratio dcf-ratio-16x9"';
+$ratioStyle = '';
+if ($type === 'audio') {
+    $ratioClass = '';
```

Comment:
Note: Once ratio for video is determined. A DCF version can be added if not currently defined and used as ratioClass here instead of inline style.